vello_shaders/src/compile/mod.rs

```diff
@@ -129,6 +130,11 @@ impl ShaderInfo {
         })
     }
 
+    /// Same as [`ShaderInfo::from_dir`] but uses the default shader directory provided by [`shader_dir`].
+    pub fn from_default() -> HashMap<String, Self> {
```

Comment:
Maybe `from_builtin`/`from_included` is a better name?

We should also point to the `SHADERS` constant as preferred by most users